src/styles/index.scss

```diff
@@ -1,3 +1,9 @@
+@import './variables';
+@import './header';
+@import './stars';
+@import './card';
+
 body {
+  display: block;
   margin: 0;
 }
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), properties within a selector should be ordered. Generally, `display` property should come after box model properties like `margin`. Consider reordering the properties to maintain consistency and readability.